schema/manager.go

```diff
@@ -347,6 +347,9 @@ func (manager *Manager) LoadSchemaFromFile(filePath string) error {
 //loadSchemaFromFile loads schema from json file - recursive version for nested schemas
 func (manager *Manager) loadSchemaFromFile(filePath string) error {
 	log.Info("Loading schema %s ...", filePath)
+	if filePath == "" {
```

Comment:
how could this happen? empty line in a config file? I wouldn't mind an automated test